### ISSUE TITLE
Possibility to configure a ltm policy  in order to reset traffic connection via connection=true and shutdown=true

### ISSUE THREAD

**Issue by diaconud007** [F5Networks/terraform-provider-bigip#463]
### Is your feature request related to a problem? Please describe.
I want to o be able to configure a ltm policy in order to reset a tcp connection via 
```
action {
      connection = true
      shutdown = true
      reset = false
      request = false
    }
```

For the moment the actions connection and shutdown are not accepted by the provider.

Using 
```
connection = true
shutdown = true
```
would be equivalent with set "Reset traffic connection at request time" in BigIP GUI

### Describe the solution you'd like
Declare those to actions in func resourceBigipLtmPolicy()  resource_bigip_ltm_policy.go and ltm.go

### Describe alternatives you've considered
Use 
```
action {
      forward = true
      reset = true
    }
```
instead, but this, even if it works, in GUI will appear "Disable asm at request time"

**Comment by nmenant** [comment]
Tracking this Feature request with INFRAANO-314

**Comment by RavinderReddyF5** [comment]
@diaconud007 fixed in v1.10.0, please check and open new issue if u see any problem.
